tests/cpp-tests/CMakeLists.txt

```diff
@@ -242,6 +249,8 @@ list(APPEND GAME_SOURCE
      Classes/ConsoleTest/ConsoleTest.cpp
      Classes/CurlTest/CurlTest.cpp
      Classes/CurrentLanguageTest/CurrentLanguageTest.cpp
+     Classes/SDKTest/SDKTest.cpp
+     Classes/PaySDKTest/PaySDKTest.cpp
```

Comment:
Can these test works in other platforms?

Reply:
And PaySDK is not part or engine, so it is not suitable to test it here.

Reply:
ok, deleted